source/agora/network/Manager.d

```diff
@@ -1366,7 +1366,8 @@ public class NetworkManager
     public void whitelist (Hash utxo)
     {
         this.peers[].filter!(p => p.utxo == utxo)
-            .each!(p => p.client.address.each!(addr => this.banman.whitelist(addr)));
+            .each!(p => p.client.address.filter!(a => a !is Address.init)
+            .each!(addr => this.banman.whitelist(addr)));
```

Comment:
Either we exclude it in the banman, or at the call site, but doing it in the banman for one and the call site for the other is a bit confusing I think. Having it in banman is more foolproof, I guess?

Reply:
It was difficult to do at call sites because it is in multiple places and difficult to see when to check. I agree it is not ideal but will enable us to move forwards.

Reply:
Updated